packages/components/src/_styles/reduceMotion.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @param {"transition" | "animation" | undefined} property
+ * @return {string}
+ */
+export function reduceMotion( property = undefined ) {
```

Comment:
@jameslnewell We did the same thing 😊 . Except I put it in a different location.
https://github.com/WordPress/gutenberg/blob/master/packages/components/src/utils/reduce-motion.js